common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -480,4 +480,13 @@ eActionOp_BML:
   ACTION_BML_CHANNEL_SCAN_DUMP_RESULTS_REQUEST: 212
   ACTION_BML_CHANNEL_SCAN_DUMP_RESULTS_RESPONSE: 213
 
-  ACTION_BML_ENUM_END: 214
+  ACTION_BML_GET_CLIENT_LIST_REQUEST: 215
```

Comment:
I think that just like we changed the `bml_set_client..` to be `bml_client_set_client..` for grouping the APIs,
we can also change the CMDUs from `ACTION_BML_GET_CLIENT...` to `ACTION_BML_CLIENT_GET_CLIENT...`

nitpick

Reply:
Changed, this makes the CMDUs more understandable